projects/ngx-prx-styleguide/src/lib/paging/paging.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, Input, Output, EventEmitter, OnChanges } from '@angular/core';
+
+@Component({
+  selector: 'prx-paging',
+  template: `
```

Comment:
TODO: This is styled with the global paging CSS and doesn't look anything like the new paging designs